### ISSUE TITLE
Read the group guideline and laws, when you are added!

### ISSUE THREAD

**Issue by diff-** [yagop/telegram-bot#219]
Is there any plugin/method to show a message to new members some messages by the bot?

**Comment by rockneurotiko** [comment]
Yeah, [service_enter_group](https://github.com/yagop/telegram-bot/blob/master/plugins/service_entergroup.lua)
Read this comment to configure the message: https://github.com/yagop/telegram-bot/pull/178#issuecomment-105698579
